src/css/sort_option.css

```diff
@@ -0,0 +1,120 @@
+.search_sort {
+  position: relative;
+  user-select: none;
+  width: 26rem;
+  font-size: 1rem;
+  font-weight: 300;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  height: 60px;
+  white-space: nowrap;
+}
+.search_sortTrigger {
+  position: relative;
+  display: grid;
+  height: 100%;
+  align-items: center;
+  padding: 0 22px;
+  color: var(--main-text-color);
+  background-color: var(--main-border-color);
+  cursor: pointer;
+}
+.search_sortTrigger::before {
+  content: '\21F5';
+  font-size: 1.5rem;
+  position: absolute;
+  padding-left: 10px;
+  margin-bottom: 5px;
+}
+@media screen and (max-width: 1024px) {
+  .search_sortTrigger {
+    width: 6ch;
+  }
+  .search_sortOptions {
+    width: max-content;
+  }
+  .search_sortLabel {
+    margin-left: 5px;
```

Comment:
This is a bit fuzzy but I couldn't find a better solution.

6ch should be 3 full-width characters, but it's actually slightly... Less. Coincidentally, I wanted a bit of space between the text and the ⇵ icon, so adding a 5px margin solved both problems...